pages/dkp/kaptain/2.0.0/install/air-gapped-2.2/index.md

```diff
@@ -16,48 +16,21 @@ You can deploy Kaptain to a cluster in a selected workspace. If you do not inten
 
 For reference values of the required number of worker nodes, CPU, RAM, and storage resources, refer to the [requirements](../requirements/) section.
 
-## DKP 2.2 air-gapped installation
-
-Refer to [DKP install instructions][dkp_install], if you want to deploy Kaptain in a networked environment or to [DKP 2.1 air-gapped instructions][2.1_air] if you are deploying with DKP 2.1.
-
-<p class="message--note"><strong>NOTE: </strong>All DKP commands in this page assume <code>KUBECONFIG=clusterKubeconfig.conf</code> is set.</p>
-
-Kaptain supports installation on an air-gapped (a.k.a. offline or private) DKP managed cluster. Before installing Kaptain, please follow the [air-gapped installation guide][konvoy-air-gap] to set up the air-gapped DKP managed cluster. The cluster admin is responsible for configuring the DKP cluster correctly and ensuring container images have been pre-loaded to the private registry before installing Kaptain.
-
 ## Prerequisites
 
-- [A DKP cluster][dkp-install] with the following Platform applications enabled:
-
-  - Istio
-  - Knative (optional, if KServe is configured to work in `RawDeployment` mode)
-
-- [`kubectl`][kubectl] on your installation machine
+Ensure you meet all [prerequisites](../prerequisites/).
 
-- For customers deploying in a multi-cluster environment (Enterprise): Ensure you have configured [Kaptain to authenticate with a Management Cluster][dex].
+<p class="message--note"><strong>NOTE: </strong>Starting from the 1.3 release, Spark Operator is no longer installed by default with Kaptain.</p>
 
-- Ensure the following applications are enabled in Kommander.
+If you need to run Spark jobs on Kubernetes using Spark Operator, you must install it separately. Use the following instructions to install Spark Operator from Kommander Catalog for [DKP 2.x][install-spark-dkp2].
 
-  Review the [Kommander installation documentation][kommander-install] for more information.
+## DKP 2.2 air-gapped installation
 
-      ```yaml
-      apiVersion: config.kommander.mesosphere.io/v1alpha1
-      kind: Installation
-      apps:
-        ...
-        dex:
-        dex-k8s-authenticator:
-        kube-prometheus-stack:
-        istio:
-        knative:
-        minio-operator:
-        traefik:
-        nvidia:  # to enable GPU support
-        ...
-      ```
+Refer to [DKP install instructions][dkp_install], if you want to deploy Kaptain in a networked environment or to [DKP 2.1 air-gapped instructions][2.1_air] if you are deploying with DKP 2.1.
 
-<p class="message--note"><strong>NOTE: </strong>Starting from the 1.3 release, Spark Operator is no longer installed by default with Kaptain.</p>
+<p class="message--note"><strong>NOTE: </strong>All DKP commands in this page assume <code>KUBECONFIG=clusterKubeconfig.conf</code> is set.</p>
```

Comment:
this note could be at the top of this page.
also are we missing the warning "Ensure the cluster that you want to use to deploy Kaptain is the only cluster in its workspace. Kaptain is meant to be deployed on workspaces with a single cluster." or is it that it doesn't apply here?

Reply:
Good catch! We actually replaced the note we had before with this one. Primarily because asking customers to "move your cluster to its own workspace" did not make sense, since we do not support dettaching and reattaching clusters yet. Deleted the ols note in favor of this one in all pages.